### ISSUE TITLE
formula API does not include a graphics module

### ISSUE THREAD

**Issue by JanSchulz** [statsmodels/statsmodels#2511]
Not sure if this is a real bug, but it was an annoyance:

Following this guid to `plot_corr` http://statsmodels.sourceforge.net/0.5.0/generated/statsmodels.graphics.correlation.plot_corr.html, the following code does not work if I substitute the formula api as `sm`:

```
import statsmodels.formula.api as sm
sm.graphics.plot_corr(corr_matrix, xnames=hie_data.names)
```
-> no module graphics

You also can't `import statsmodels.api.graphics as smg` :-(

In the end I used:
```
import statsmodels.api
smg = statsmodels.api.graphics
```

**Comment by josef-pkt** [comment]
I was trying to promote:

```
import statsmodels.api as sm
import statsmodels.formula.api as smf
```

but IIRC, then `sm.formula.xxx` should also work if sm is statsmodels.api

for non-interactive use the best is to use correct module import paths, e.g.
`import statsmodels.graphics.api as smg` (I guess)

Most of this is by design or a side effect of the api design, and I don't know if we can or should change anything. We won't fill up the statsmodels.formula.api namespace with non formula functions.

**Comment by JanSchulz** [comment]
Yes, `import statsmodels.graphics.api as smg` works.

**Comment by JanSchulz** [comment]
merged in #2512
